### DISCUSSION TITLE
Less not supported

### DISCUSSION THREAD

**Discussion by wekor** [remix-run/remix#3382] [Proposals]
We have many encapsulated components that use less. This framework does not support less, so we need to renovate all components，This is a very bad phenomenon，React has many ecosystems using less。

**Comment by sergiodxa** [comment]
You can keep using LESS, but you'll have to setup a separate process to take your LESS files and convert them to CSS, then in the Remix app import the generated CSS files, this is the same approach used to support SASS/SCSS, Tailwind and PostCSS in general.

**Reply by wekor** [reply]
How to move items? How to distinguish between so many CSS names? There are many beautiful components that use less. How can I introduce them?
